flink-core/src/main/java/org/apache/flink/api/common/io/FileInputFormat.java

```diff
@@ -819,6 +819,10 @@ public void open(FileInputSplit fileSplit) throws IOException {
 			this.stream = isot.waitForCompletion();
 			this.stream = decorateInputStream(this.stream, fileSplit);
 		}
+		catch (FileNotFoundException e) {
+			throw (FileNotFoundException)(new FileNotFoundException("Input split " + fileSplit.getPath() +
```

Comment:
I don't understand why "skip and continue" is in this message.
Not all users of the `FileIputFormat` skip and continue. The interpretation of the exception should not be assumed when creating the exception.